react-app/src/native/components/leagues/TeamList.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Text, Card, CardItem} from 'native-base';
+import { Table, Row } from 'react-native-table-component';
+import { StyleSheet } from 'react-native';
+import Loading from '../common/Loading';
+
+export default class Schedule extends React.Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+
+        const { league, leagueName, lookups } = this.props;
+
+        const flexArr = [2, 7];
+        const teamTable = {
+            header: ['Team #', 'Name'],
+            data: [],
+        }
+
+        league.teams.map((team, i) => {
+            teamTable.data.push([team.numInLeague, team.name]);
+        });
+
+        if (league == null || league.isFetching) return <Loading />
+
+        return (
+            <Card>
+                <CardItem header>
+                    <Text>{leagueName}</Text>
+                </CardItem>
+                <CardItem cardBody style={{ padding: 10 }}>
+                    <Table style={{ flex: 1, marginBottom: 10 }} borderStyle={{ borderWidth: 0, borderColor: "transparent" }}>
```

Comment:
These styles should be included as part of your styles object below. Try to have as little inline styling as possible, it's just easier for future editing